src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The 'href' attribute in the link tag should point to a CSS file, not an SCSS file. Please ensure the path is correct and points to the compiled CSS file.